IndicoV2.V1Adapters.Tests/Converters/SubmissionStatusConverterTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Linq;
+
+using FluentAssertions;
+
+using IndicoV2.V1Adapters.Converters;
+
+using NUnit.Framework;
+
+using V1Status = Indico.Types.SubmissionStatus;
+using V2Status = IndicoV2.Submissions.Models.SubmissionStatus;
+
+
+namespace IndicoV2.V1Adapters.Tests.Converters
+{
+    public class SubmissionStatusConverterTests
+    {
+        private static object[][] _v1StatusToV2StatusMap = Enum.GetValues(typeof(V1Status))
```

Comment:
It's missing cases when There's V2 but no matching V1 status.

Reply:
I have rewritten these tests.